python/cudf/cudf/core/column/string.py

```diff
@@ -1574,16 +1574,20 @@ def endswith(self, pat, na=np.nan, **kwargs):
             pattern matches the end of each string element.
 
         """
-        if na is not np.nan:
-            raise NotImplementedError("`na` parameter is not yet supported")
+        na = kwargs.get("na", None)
+        if na is not None:
+            warnings.warn(
+                "`na` parameter is not yet supported, \
+                as input series is expected to have true strings"
+            )
```

Comment:
Would change this to:
```suggestion
        if "na" in kwargs:
            warnings.warn(
                "`na` parameter is not supported, \
                as cudf uses native strings instead of Python objects"
            )
```

Reply:
modified 👍